Assets/MixedRealityToolkit/Utilities/Physics/TwoHandScaleLogic.cs

```diff
@@ -17,28 +17,35 @@ namespace Microsoft.MixedReality.Toolkit.Core.Utilities.Physics
     /// </summary>
     public class TwoHandScaleLogic
     {
-        private Vector3 startObjectScale;
-        private float startHandDistanceMeters;
+        private Vector3 m_startObjectScale;
```

Comment:
We've tended to not use prefixes for variable names. Please revert these prefix additions, then I think this PR looks good!

Reply:
Updated!